benchmark/build.gradle.kts

```diff
@@ -0,0 +1,36 @@
+plugins {
+    scala
+    kotlin("jvm") version "1.4.10"
+    id("cz.alenkacz.gradle.scalafmt") version "1.16.2"
+}
+
+group = "org.virtuslab"
+version = "1.0-SNAPSHOT"
+
+repositories {
+    mavenCentral()
+    maven("https://repo.kotlin.link")
+    maven("https://maven.pkg.jetbrains.space/kotlin/p/kotlin/kotlin-js-wrappers")
+}
+
+val scalaVersion = "_2.13"
```

Comment:
Nit: (if you're sticking to Gradle), just use `2.13` and include the `_` explicitly in Maven coords below for readability